models/migrations/v1_19/v241.go

```diff
@@ -0,0 +1,15 @@
+// Copyright 2023 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package v1_19 //nolint
+
+import (
+	"code.gitea.io/gitea/models/db"
+
+	"xorm.io/xorm"
+)
+
+func CreateRepoSettingsTable(x *xorm.Engine) error {
+	type RepoSetting db.ResourceSetting // to generate a table named repo_setting
```

Comment:
For DB migrations please copy struct into the migration so that if it changes in the future the migration still works the same

Reply:
Done